src/MonitorProfiler/Utilities/OomUtilities.h

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
```

Comment:
Move all of these to https://github.com/dotnet/dotnet-monitor/blob/main/src/inc/macros.h; these feel common enough that they don't need their own special header file.